From f6dbe616669f25ca0b54285f0cbd1f7f824b72d1 Mon Sep 17 00:00:00 2001
From: harsh4136 <patelharshn2012@gmail.com>
Date: Fri, 14 Jun 2024 20:48:16 +0530
Subject: [PATCH 2/3] add third line

---
 index.html | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/index.html b/index.html
index e69de29..ef30eb4 100644
--- a/index.html
+++ b/index.html
@@ -0,0 +1,3 @@
+<p>hello world</p>
+<p>edit by harsh</p>
+<p>edit third line</p>
\ No newline at end of file
-- 
2.45.0

